modules/tinymce/src/plugins/codesample/test/ts/module/CodeSampleTestUtils.ts

```diff
@@ -9,8 +9,8 @@ const dialogSelector = 'div.tox-dialog';
 const toolbarButtonSelector = 'button[aria-label="Insert/edit code sample"]';
 
 const setLanguage = (newLanguage: string): void => {
-  const select = document.querySelector('div[role="dialog"] select') as HTMLSelectElement;
-  select.value = newLanguage;
+  const select = document.querySelector('div[role="dialog"] div[class="tox-listboxfield"] button') as HTMLSelectElement;
```

Comment:
This `as` is wrong, it's probably now a button element?

It won't impact the code below, but we need to make sure our casts are accurate

Reply:
```suggestion
  const select = document.querySelector('div[role="dialog"] div[class="tox-listboxfield"] button') as HTMLElement;
```